next/next.config.js

```diff
@@ -4,7 +4,13 @@ const { i18n } = require("./next-i18next.config");
 const nextConfig = {
   reactStrictMode: true,
   images: {
-    domains: [process.env.NEXT_IMAGE_DOMAIN],
+    remotePatterns: [
+      {
+        protocol: "https",
+        hostname: process.env.NEXT_IMAGE_DOMAIN,
+        pathname: "**",
+      },
+    ],
```

Comment:
The domains setting was deprecated, and generated a warning when running the site.

@joshua-scott this happens in your project as well I think, maybe you want to do this same change?